ts/private/ts_project.bzl

```diff
@@ -86,18 +86,23 @@ See https://github.com/aspect-build/rules_ts/issues/228 for more details.
     arguments.add_all(ctx.attr.args)
 
     outdir = _lib.join(ctx.label.package, ctx.attr.out_dir) if ctx.attr.out_dir else ctx.label.package
+    if ctx.label.workspace_root:
+        outdir = _lib.join(ctx.label.workspace_root, outdir)
     if outdir == "":
         outdir = "."
+    tsconfig_path = ctx.file.tsconfig.path.removeprefix("%s/" % ctx.bin_dir.path)
```

Comment:
Can `lib.relative_to_package` be used for this?

Reply:
I don't think so because that strips the `external/${workspaceName}/` prefix which seems to be required and I think that's desired in this case? I'm not totally sure how external targets are actually laid out in the temporary directory, but using `.removeprefix()` the path is `extenral/lib_wksp/tsconfig.json` while using `lib.relative_to_package()` just uses `tsconfig.json`. I think we want the former so we need to stick with `.removeprefix()`?